src/inference/src/cnn_network_ngraph_impl.cpp

```diff
@@ -75,7 +75,8 @@ void CNNNetworkNGraphImpl::createDataForResult(const ::ngraph::Output<::ngraph::
     // query shape from ngraph::Parameter output shape and check there are no zeros in it
     for (const auto& dim : shape) {
         if (dim.is_static() && dim.get_length() == 0)
-            IE_THROW() << outName << " has zero dimension which is not allowed";
+            continue;
```

Comment:
The following checks are problematic when it comes to returning the ShapeOf scalar - it raises an error `aten::_shape_as_tensor_1 has zero dimension which is not allowed`. Commenting out both these checks seems to solve the issue, but I couldn't find a workaround